README.md

```diff
@@ -0,0 +1,161 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Video Presentation](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<br/>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🏫 Ruby Capstone Project <a name="about-project"></a>
+
+In this project, we are building a item library app using Ruby. The application allows managing items and store them.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Programming Language</summary>
+  <ul>
+   <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Books Games and Music Albums:** Ability to add new items to the system.
+- **Features Management:** Manage a list of features like authors,genres and labels in the library.
+- **Save items & features:** Save you items and features created in json files.
+- **Modular Design:** Built with Object-Oriented Programming (OOP) principles for easy scalability and maintenance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Video Presentation <a name="live-demo"></a>
+- [Project Presentation Video](https://drive.google.com/file/d/1GSmMYJk5fQkSfFdOr96AuVywlGVRo3As/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please kindly  highlight some interesting piece of code or something you built that you are very proud of in the video presentation as stated in the requirement.

<img width="649" alt="Untitled" src="https://github.com/Felipe-Perez-Ferraro/RUBY_Capstone/assets/103386183/44d42625-4747-4f4b-9fe9-2dbf4a679f58">